src/lib.rs

```diff
@@ -288,6 +324,37 @@ where
             result
         }
     }
-    
+
     |f| IO::new(Bracket (acquire, release, Some (f)))
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    /// Test whether [`bracket`] sequences actions correctly.
+    fn sequencing_bracket () {
```

Comment:
Would a name like `bracket_sequences_actions_correctly` be better here? 
I'm basing this entirely off of cargo naming the default test function when you generate a project with `cargo new --lib` "`it_works`"

Reply:
That seems to be the test naming conventions that the rust stdlib sometimes follows(but it's inconsistent about it)